src/FlightDisplay/FlyViewWidgetLayer.qml

```diff
@@ -49,233 +51,843 @@ Item {
+            font.pointSize: 30
+            text: _efficiency
+        }
+        height: parent.height / 3
+        width: parent.width / 4
+        color: "transparent"
+        border.color: "black"
     }
 
-    MapScale {
-        id:                 mapScale
-        anchors.margins:    _toolsMargin
-        anchors.left:       toolStrip.right
-        anchors.top:        parent.top
-        mapControl:         _mapControl
-        buttonsOnLeft:      false
-        visible:            !ScreenTools.isTinyScreen && QGroundControl.corePlugin.options.flyView.showMapScale && mapControl.pipState.state === mapControl.pipState.fullState
-
-        property real centerInset: visible ? parent.height - y : 0
+    // Nav
+    Rectangle{
+        id: nav
+        anchors{
+            left: efficiency.right
+            top: stats.bottom
+        }
+        Text {
+            id: navTitle
+            text: qsTr("Nav")
+            anchors{
+                top: parent.top
+                left: parent.left
+                right: parent.right
+            }
+            horizontalAlignment: Text.AlignHCenter
+            font.pointSize: 20
+            font.underline: true
+        }
+        Text {
+            id: navDeg
+            anchors{
+                top: navTitle.bottom
+                left: parent.left
+                right: parent.right
+            }
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+            font.bold: true
+            font.pointSize: 30
+            text: _heading
+        }
+        Text {
+            anchors{
+                top: navDeg.bottom
+                bottom: parent.bottom
+                left: parent.left
+                right: parent.right
+            }
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+            font.bold: true
+            font.pointSize: 30
+            text: {
+                if (_heading != ' ') {
+                    var a = _heading % 45
+                    switch(_heading - a) {
+                        case 0:     return "N"
+                        case 45:    return "NE"
+                        case 90:    return "E"
+                        case 135:   return "SE"
+                        case 180:   return "S"
+                        case 225:   return "SW"
+                        case 270:   return "W"
+                        case 315:   return "NW"
+                    }
+                    return "ERROR"
+                }
+                else {
+                    return ' '
+                }
+                 
+            }
+        }
+        height: parent.height / 3
+        width: parent.width / 4
+        color: "transparent"
+        border.color: "black"
+        
     }
 
     Component {
         id: preFlightChecklistPopup
         FlyViewPreFlightChecklistPopup {
         }
     }
+
+    
+
+
+
+
+
+
+//     QGCToolInsets {
```

Comment:
I guess all of this commented out code is what was there before?  Might as well get rid of it.  Don't think it serves any value.

Reply:
Removed.